README.md

```diff
@@ -5,19 +5,57 @@
   <br>
 </h1>
 
-A Hackathon HeartBits de 2018 é um evento de organização conjunta da Associação
+The Hackathon HeartBits of 2018 is an event organized jointly by the National
+Association of Medical Students (ANEM), Computer Engineer Student Group from
```

Comment:
Perhaps linking to the associations' pages with markdown? 

Also CeSIUM is Software Engineering, not Computer 😛

Reply:
Okay, I am on it! :+1: